staging/src/k8s.io/pod-security-admission/manifests/validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: "webhook.podsecurity-webhook.svc.cluster.local"
+webhooks:
+  - name: "webhook.podsecurity-webhook.svc.cluster.local"
+    rules:
+      - apiGroups: [""]
+        apiVersions: ["v1"]
+        operations: ["CREATE", "UPDATE"]
+        resources:
+          - cronjobs
+          - daemonsets
+          - deployments
+          - jobs
+          - namespaces
+          - pods
+          - pods/ephemeralcontainers
+          - podtemplates
+          - replicasets
+          - replicationcontrollers
+          - statefulsets
```

Comment:
the resources need to be split by API group:

```yaml
      - apiGroups: [""]
        apiVersions: ["v1"]
        operations: ["CREATE", "UPDATE"]
        resources:
          - namespaces
          - pods
          - pods/ephemeralcontainers
          - podtemplates
          - replicationcontrollers
      - apiGroups: ["apps"]
        apiVersions: ["v1"]
        operations: ["CREATE", "UPDATE"]
        resources:
          - daemonsets
          - deployments
          - replicasets
          - statefulsets
      - apiGroups: ["batch"]
        apiVersions: ["v1"]
        operations: ["CREATE", "UPDATE"]
        resources:
          - cronjobs
          - jobs
```

Reply:
consider splitting the advisory-only rules (for apps and batch API groups, and podtemplates and replicationcontrollers) into a second webhook definition (in the same ValidatingWebhookConfiguration object) that has a failurePolicy of `Ignore` (we'd rather permit write requests to those if our webhook has a problem, since we'd only be warning, not enforcing)